components/tools/ToolsList.js

```diff
@@ -6,7 +6,7 @@ export default function toolsList({ toolsData }) {
     <div className="" data-testid="ToolsList-main" >
       {Object.keys(toolsData).map((categoryName, index) => {
         if(toolsData[categoryName].toolsList.length > 0) return (
-        <div className='my-8' key={index} id={categoryName}>
+        <div className='my-8 scroll-m-[100px]' key={index} id={categoryName}>
```

Comment:
Don't use units in `px`. Instead use `rem`, `em` or tailwind utility classes to add a scroll margin. Using pixels will impact responsiveness of the website.